cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java

```diff
@@ -66,6 +68,13 @@ public Cocos2dxWebView(Context context, int viewTag) {
 
         this.setWebViewClient(new Cocos2dxWebViewClient());
         this.setWebChromeClient(new WebChromeClient());
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
Yes, it will cause a compilation error. In fact this PR causes two compilation errors. Both errors are related to the version of SDK used for compilation. I wrote about it above.

In conclusion, this PR shouldn't be merged until someone will bump the targetSdkVersion for Ant builds (Gradle builds are using targeSdkVersion == 22).